### PR TITLE
Test report

### PR DESCRIPTION
**Pull request by Daniel2894**

### CODE REVIEW

**Comment by Daniel2894** [comment]
Me falto la imagen del reporte, aquí está:
mochawesome report

File: report/ApiTesting.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...ot;rootEmpty&quot;:true,&quot;_timeout&quot;:5000},&quot;copyrightYear&quot;:2018}" data-config="{&quot;reportFilename&quot;:&quot;ApiTesting&quot;,&quot;reportDir&quot;:&quot;report&quot;,&quot;reportTitle&quot;:&quot;workshop-api-testing-js&quot;,&quot;reportPageTitle&quot;:&quot;Mochawesome Report&quot;,&quot;inline&quot;:false,&quot;inlineAssets&quot;:false,&quot;cdn&quot;:false,&quot;charts&quot;:true,&quot;enableCharts&quot;:true,&quot;code&quot;:true,&quot;enableCode&quot;:true,&quot;autoOpen&quot;:false,&quot;overwrite&quot;:true,&quot;timestamp&quot;:false,&quot;ts&quot;:false,&quot;showPassed&quot;:true,&quot;showFailed&quot;:true,&quot;showPending&quot;:true,&quot;showSkipped&quot;:false,&quot;showHooks&quot;:&quot;failed&quot;,&quot;saveJson&quot;:true,&quot;saveHtml&quot;:true,&quot;dev&quot;:false,&quot;assetsDir&quot;:&quot;report\\assets&quot;,&quot;jsonFile&quot;:&quot;C:\\Users\\drestrepom\\Documents\\Projects\\workshop-api-testing-js\\report\\ApiTesting.json&quot;,&quot;htmlFile&quot;:&quot;C:\\Users\\drestrepom\\Documents\\Projects\\workshop-api-testing-js\\report\\ApiTesting.html&quot;}"><div id="report"></div><script src="assets\app.js"></script></body></html>
```
Comment (sorregom) [inline report/ApiTesting.html]: EOF

File: report/ApiTesting.json [hunk 2]
```diff
@@ -0,0 +1,187 @@
+{
```
Comment (sorregom) [inline report/ApiTesting.json]: please check again you are pushing reports
you can use the rebase to make this change a single commit after deleting the report, then a git push -f to rewrite your branch to never have this on its commits

**Review by sorregom — CHANGES_REQUESTED** [review]
check again, you are pushing reports

**Review by sorregom — APPROVED** [review]
LGTM

**Pull request merged**